### DISCUSSION TITLE
Why TabView need absolute positioning?

### DISCUSSION THREAD

**Discussion by byStrange** [react-native-elements/react-native-elements#3897] [Q&A]
I was using the Tab component from RNE and there was a problem when I  was trying to implement scrolling, I have this kind of architecture in the profile screen
UserProfileCard
Tab (new posts, blocked posts)
TabView (user posts) 

and you should be able to scroll on this profile screen. I wrapped everything with ScrollView but it did not work because TabView components have absolute positioning that is why they don't take any height from the screen, to fix that I should give height manually which is not a good approach in my opinion. I'm not an expert in react native but from what I know as a developer when absolute positioning is used it doesn't take any height or width from the screen. I was reading your TabView.js code in  base/dist/TabView.js and there I saw a line where you gave Stylesheet.absoluteFillObject, and if I remove that scrolling works since now TabView takes up space which is what I need. Is it good or is there any way to make my Screen scrollable?

**Comment by rasika-v-23** [comment]
yes facing same issue
